internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -227,10 +258,10 @@ func (opts *Opts) askClusterOptions() error {
 
 	clusterName := opts.ClusterName
 	if clusterName == "" {
-		message := "Insert the cluster name"
+		message := "Cluster Name:"
 		clusterName = opts.newClusterName()
 		if clusterName != "" {
-			message = fmt.Sprintf("Insert the cluster name [Press Enter to use the auto-generated name '%s']", clusterName)
+			message = fmt.Sprintf("Cluster Name [Press Enter to use the auto-generated name '%s']:", clusterName)
```

Comment:
[nit] in the `prompt` file where you do 
```go
Prompt: &survey.Input{
	Message: message,
```

You can instead fo 
```go
Prompt: &survey.Input{
	Message: fmt.Sprintf("Cluster Name%s:", defaultValue),
```

And pass the default as an extra string to the method, this may be a bit cleaner on re using parts of the string

ditto for the other inputs with default